pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -487,7 +487,7 @@ func NewControllerDeployment(namespace, repository, imagePrefix, controllerVersi
 
 // Used for manifest generation only
 func NewOperatorDeployment(namespace, repository, imagePrefix, version, verbosity, kubeVirtVersionEnv, virtApiShaEnv, virtControllerShaEnv, virtHandlerShaEnv, virtLauncherShaEnv, virtExportProxyShaEnv,
-	virtExportServerShaEnv, gsShaEnv, virtApiImageEnv, virtControllerImageEnv, virtHandlerImageEnv, virtLauncherImageEnv, virtExportProxyImageEnv, virtExportServerImageEnv, gsImage,
+	virtExportServerShaEnv, gsShaEnv, runbookURLTemplate, virtApiImageEnv, virtControllerImageEnv, virtHandlerImageEnv, virtLauncherImageEnv, virtExportProxyImageEnv, virtExportServerImageEnv, gsImage,
```

Comment:
I think it deserves a struct please
too many arguments

Reply:
@oshoval Hi, thank you for the notice! Can we have this refactoring in a separate PR? As I see it, the refactoring of that will occur in multiple places (I was thinking to move all the supported env vars into the env var manager). If you will agree, I will create a refactoring PR as soon as that one will get in.

Reply:
@oshoval I had a cognitive load while trying to create some struct and populate it. I came into conclusion that it's create some map of strings and try to utilize it, or even via the env var manager. What I am trying to say is that an intermediate solution here will also be a boilerplate.

Reply:
sure np